internal/server/statvar/statvar_group.go

```diff
@@ -114,6 +116,51 @@ func filterSVG(in *pb.StatVarGroups, statVars []string) *pb.StatVarGroups {
 	return result
 }
 
+// adjustDescendentStatVarCount returns the DescendentStatVarCount for node curID.
+// Mutates DescendentStatVarCount for all nodes in id2Node starting at customSvgRoot.
+func adjustDescendentStatVarCount(id2Node map[string]*pb.StatVarGroupNode, curID string) int32 {
+	curNode, ok := id2Node[curID]
+	if !ok {
+		return 0
+	}
+	curNode.DescendentStatVarCount = int32(len(curNode.ChildStatVars))
+	for _, childSVG := range curNode.GetChildStatVarGroups() {
+		// Child SVG protos have its own DescendentStatVarCount that must be set.
+		childSVG.DescendentStatVarCount = adjustDescendentStatVarCount(id2Node, childSVG.GetId())
+		curNode.DescendentStatVarCount += childSVG.DescendentStatVarCount
+	}
+	return curNode.GetDescendentStatVarCount()
+}
+
+// mergeSVGNodes merges node2's svg and svs into node1.
+// Merge here refers to set operation.
+// Warning: node1's DescendentStatVarCount is unchanged.
+// Caller is responsible for calling fixCustomRootDescendentStatVarCount
```

Comment:
Where is fixCustomRootDescendentStatVarCount, or you mean adjustDescendentStatVarCount?

Reply:
Oops I need to update the comment, it should be adjustDescendentStatVarCount